lib/msal-angular/src/msal.broadcast.service.ts

```diff
@@ -22,4 +22,7 @@ export class MsalBroadcastService {
             this._msalSubject.next(message);
         });
     }
+    broadcast(message: EventMessage) {
```

Comment:
Can you please add a return type?
```suggestion
    broadcast(message: EventMessage): void {
```